msal/build.gradle

```diff
@@ -277,7 +277,14 @@ publishing {
 }
 
 tasks.whenTaskAdded { task ->
-    if (task.name == 'assembleDebug' || task.name == 'assembleRelease') {
-        task.dependsOn 'checkstyle', 'pmd', 'lint', 'javadocJar', 'sourcesJar'
+    if (task.name.contains('assemble')) {
+        task.dependsOn 'checkstyle', 'pmd'
+    }
+
+    if (task.name.contains('assemble')
+        && !task.name.contains('Snapshot')
+        && !task.name.contains('Test')
+        && !task.name.contains('Local')) {
```

Comment:
Don't we want lint for local?

Reply:
See commit message for da2fc37bfbca71501f556f8baaa723d1df72d0a9

Running `lint` was also running `dist` tasks for reasons I didn't fully troubleshoot

Reply:
(Try it locally, you'll see a `dist` build kickoff and subsequently fail)